src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,99 @@
+package duke;
+
+import java.util.Scanner;
+
+
+import java.util.ArrayList;
+import java.io.IOException;
+
+ 
+
+
+
+public class Duke {
+    public static void main(String[] args) throws IOException, ClassNotFoundException {
+
+
+     
+        //print welcome message, ask for user input
+        System.out.println("  ───── ･ ｡ﾟ☆: *.☽ .* :☆ﾟ. ───── \n Hello! I'm Broccoli the dinosaur \n           <|°▿▿▿▿°|/ \n      What can I do for you? \n   ──── ･ ｡ﾟ☆: *.☽ .* :☆ﾟ. ────" ); 
+        //getInput();
+        
+
+
```

Comment:
Perhaps we can remove some empty lines and used codes to make the program look neater. :grinning: